dataScience/src/main/java/edu/csula/datascience/aquisition/deathCollectorRecord.java

```diff
@@ -0,0 +1,120 @@
+package edu.csula.datascience.aquisition;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+
+import com.mongodb.BasicDBObject;
+
+public class deathCollectorRecord extends mongoDB implements Collector<List<String>, List<String>> {
+
+	public Collection<List<String>> mungee(Collection<List<String>> src) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	public void save(String path, String mongoDBCollection) {
+		// TODO Auto-generated method stub
+
+		String csvFile = path;
+		BufferedReader br = null;
+		String line1 = "";
+		String cvsSplitBy = ",";
+
+		try {
+
+			br = new BufferedReader(new FileReader(csvFile));
+			while ((line1 = br.readLine()) != null) {
+
+				BasicDBObject document = new BasicDBObject();
+
+				String[] Agetype = line1.split(cvsSplitBy);
+				if (csvFile == "DeathRecords.csv") {
+					if (Agetype[7] == "") {
+						Agetype[7] = "N/A";
+					}
+					if (Agetype[8] == "") {
+						Agetype[8] = "N/A";
+					}
+					if (Agetype[10] == "") {
+						Agetype[10] = "N/A";
+					}
+					if (Agetype[14] == "") {
+						Agetype[14] = "N/A";
+					}
+					if (Agetype[15] == "") {
+						Agetype[15] = "N/A";
+					}
+					if (Agetype[17] == "") {
+						Agetype[17] = "N/A";
+					}
+					if (Agetype[19] == "") {
+						Agetype[19] = "N/A";
+					}

```

Comment:
All these if statement should really be under munge method.

Second of all, you cannot use `==` to compare String field. You have to use `.equals`. Have you guys tested your own code?